templates/mon_users.sql.j2

```diff
@@ -0,0 +1,2 @@
+CREATE USER mon_persister IDENTIFIED BY '{{ vertica_users.mon_persister }}';
```

Comment:
There is no mention anywhere of how these variables should be setup. I think the usernames shouldn't be hardcoded but rather part of the variable.